graphql_api/types/test_analytics/test_analytics.graphql

```diff
@@ -0,0 +1,31 @@
+"""
+TestAnalytics are fields related to Codecov's Test Analytics product offering
+"""
+type TestAnalytics {
+  "Results are test analytics data per test"
+  results(
+    filters: TestResultsFilters
+    ordering: TestResultsOrdering
+    first: Int
+    after: String
+    last: Int
+    before: String
+  ): TestResultConnection! @cost(complexity: 10, multipliers: ["first", "last"])
+
+  "Results aggregates are test analytics data totals across all tests"
+  resultsAggregates: TestResultsAggregates
+
+  "Flake aggregates are flake totals across all tests"
+  flakeAggregates: FlakeAggregates
+}
+
+type TestResultConnection {
```

Comment:
`TestResultConnection` and `TestResultEdge` types are moved over from the `repository.graphql` file